save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestExecutionService.kt

```diff
@@ -0,0 +1,32 @@
+package org.cqfn.save.backend.service
+
+import org.cqfn.save.backend.repository.TestExecutionRepository
+import org.cqfn.save.entities.TestExecution
+import org.springframework.stereotype.Service
+
+/**
+ * Service for test result
+ */
+@Service
+class TestExecutionService(private val testResultRepository: TestExecutionRepository) {
+    /**
+     * @param testExecutions list of test executions
+     */
+    fun saveTestResult(testExecutions: List<TestExecution>) {
+        testExecutions.forEach { testExec ->
+            testResultRepository.run {
```

Comment:
I think `run` is redundant here, because `testResultRepository` already is non-nullable and immutable, so scope function doesn't give any benefits